edx_management_commands/management_commands/management/commands/put_user.py

```diff
@@ -0,0 +1,105 @@
+"""
+Management command `put_group` is used to idempotently create Django groups
+and set their permissions by name.
+"""
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.translation import gettext as _
+
+
+@transaction.atomic
+class Command(BaseCommand):
+    # pylint: disable=missing-docstring
+
+    help = 'Creates the specified user, if it does not exist, and sets its groups.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+        parser.add_argument('email')
+        parser.add_argument('--superuser', dest='is_superuser', action='store_true')
```

Comment:
I have not gone ahead and added this, because I'd rather make it in a subsequent change and it seems a little YAGNI.  superuser and staff are the standard Django permission bits.